src/rpc/misc.cpp

```diff
@@ -135,6 +135,12 @@ class DescribeAddressVisitor : public boost::static_visitor<UniValue>
             int nRequired;
             ExtractDestinations(subscript, whichType, addresses, nRequired);
             obj.push_back(Pair("script", GetTxnOutputType(whichType)));
+            CKeyID keyID;
+            CPubKey pubkey;
+            if (pwalletMain->GetWitnessKeyID(scriptID, keyID) &&
+                pwalletMain->GetPubKey(keyID, pubkey) && pubkey.IsCompressed()) {
```

Comment:
True an assert may make the most sense here. It really shouldn't happen, and if it is we want to find out as soon as possible to avoid loss of funds.